### DISCUSSION TITLE
Question: Running with --skip-tags/--tags

### DISCUSSION THREAD

**Discussion by markush81** [semaphoreui/semaphore#2499] [Q&A] [answered]
### Question

Following up https://github.com/semaphoreui/semaphore/issues/832 i have trouble running with `--skip-tags/--tags` and it seems a lot changed in the past.

What i tried ...

![image](https://github.com/user-attachments/assets/be71a4e9-ebd4-4912-8f76-c7210d8303ba)

But this leads to ...

```
3:06:58 PM
No /var/data/semaphore_work/repository_1_7/roles/requirements.yml file found. Skip galaxy install process.
3:06:58 PM
ERROR! the playbook: --skip-tags base could not be found
```

And i wonder if the order of appends is correct https://github.com/semaphoreui/semaphore/blob/develop/services/tasks/LocalJob.go#L385

**Comment by markush81** [comment]
seems to be same as https://github.com/semaphoreui/semaphore/issues/2367

**Comment by Omicron7** [comment, marked as answer]
I've had trouble with args containing spaces. Try `--skip-tags=base` and `--tags=ansible-env`

**Comment by markush81** [comment]
> I've had trouble with args containing spaces. Try `--skip-tags=base` and `--tags=ansible-env`

Thanks, that works ... also weird, that it works on command line each way.

**Comment by fiftin** [comment]
@markush81 it is because your example similar to `"--skip-tags base" "--tags ansible-env"`.
Correct example:

![image](https://github.com/user-attachments/assets/fddb568f-f8c4-4ef5-8646-9ddf37dca9bb)

If you have any ideas on how to improve UX, I'd love to hear them ;)
